week7/answers.sql

```diff
@@ -1,3 +1,256 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA destruction;
 USE destruction;
+
+CREATE TABLE players
+(
+  player_id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(30),
+  last_name VARCHAR(30),
+  email VARCHAR(30)
+);
+
+CREATE TABLE characters 
+(
+    character_id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    player_id INT,
+    name VARCHAR(255),
+    level INT,
+    FOREIGN KEY (player_id) REFERENCES players(player_id)
```

Comment:
FOREIGN KEY CONSTRAINTs should have ON UPDATE and ON DELETE clauses assigned... this is actually required for getting expected results in this assignment.

I HIGHLY recommend naming your CONSTRAINTs.